libs/pika/async_mpi/src/mpi_polling.cpp

```diff
@@ -866,9 +853,10 @@ namespace pika::mpi::experimental {
         auto mode = get_completion_mode();
         if (mode >= pika::detail::to_underlying(detail::handler_method::unspecified))
         {
-            PIKA_THROW_EXCEPTION(
-                pika::error::invalid_status, "Bad completion flags", "invalid completion mode");
+            // PIKA_THROW_EXCEPTION(
+            //     pika::error::invalid_status, "Bad completion flags", "invalid completion mode");
```

Comment:
Why is this disabled?

Reply:
That was me playing with dlaf and using the 'old' mode which can be removed and this put back